docker/ci/install-mysql.bash

```diff
@@ -15,16 +15,22 @@
 
 set -e
 
-apt-get -qq update 
-
-apt-get install -y libmysqlclient-dev > /dev/null
-
-# Install MySQL server without a password prompt
-echo 'mysql-server mysql-server/root_password password root' | debconf-set-selections
-echo 'mysql-server mysql-server/root_password_again password root' | debconf-set-selections
-apt-get -qq update && apt-get install -y mysql-server > /dev/null
+echo "Install MySQL server without a password prompt ..."
+echo 'mysql-server mysql-server/root_password password root' | \
+    debconf-set-selections
+echo 'mysql-server mysql-server/root_password_again password root' | \
+    debconf-set-selections
+apt-get install -y mysql-server > /dev/null
 mkdir -p /var/run/mysqld
 mkdir -p /var/lib/mysql
 chown mysql:mysql /var/run/mysqld
 chown mysql:mysql /var/lib/mysql
 mkdir -p /docker-entrypoint-initdb.d
+
+echo "Install MySQL client library in C and Python ..."
```

Comment:
I moved the installation of MySQL clients from `install-python.bash` into this file `install-myql.bash`.